internal/backend/logger/logger.go

```diff
@@ -13,7 +13,7 @@ import (
 
 type Config struct {
 	Zap      zap.Config `koanf:"zap"`
-	Level    int        `koanf:"level"`    // -1 = Debug, 0 = Info, etc.
+	Level    string     `koanf:"level"`    // Lower-case or all caps, default = "info".
```

Comment:
Are we OK with breaking schema change? If not maybe call it `LevelName` and keep `Level` around.

Reply:
Excellent point. I forgot to mention this in the PR title & description.

Originally I thought about this as a breaking change, but I can also accept numbers (as before) as a temporary fallback.